Author: Cesare Falco <c.falco@ubuntu.com>
Description: Fix mame.6 and jedutil.1, update chdman.1. Changed e-mail address on all man pages.
Forwarded: yes

Index: mame-0.147u3/src/osd/sdl/man/mame.6
===================================================================
--- mame-0.147u3.orig/src/osd/sdl/man/mame.6	2012-09-18 22:35:07.000000000 +0200
+++ mame-0.147u3/src/osd/sdl/man/mame.6	2012-12-02 02:12:13.000000000 +0100
@@ -6,14 +6,14 @@
 .\" * commands: see src/emu/clifront.c clifront.h
 .\" * options: core entries, see src/emu/emuopts.c emuopts.h
 .\"            SDL-specific entries, see src/osd/sdl/sdlmain.c osdsdl.h
-.\" Cesare Falco <cesare.falco@gmail.com>, February 2007
+.\" Cesare Falco <c.falco@ubuntu.com>, February 2007
 .\"
 .\" Also, some text borrowed from the xmame 0.106 man page,
 .\" done by Rene Herrmann <rene.herrmann@berlin.de>, September 2000
 .\" and updated by Andrew Burton <burtona@gol.com>, July 2003
 .\"
 .\"
-.TH MAME 1 2012-03-25 0.145u5 "MAME \- The Multiple Arcade Machine Emulator"
+.TH MAME 6 2012-12-01 0.147u3 "MAME \- The Multiple Arcade Machine Emulator"
 .\"
 .\"
 .\" NAME chapter
@@ -56,7 +56,7 @@
 .\" *******************************************************
 .TP
 .B \-createconfig, \-cc
-Creates the default \fBmame.ini\fP file in the current directory. All the
+Creates the default \fBmame.ini\fR file in the current directory. All the
 configuration options (not commands) described below can be permanently
 changed by editing this configuration file.
 .TP
@@ -65,15 +65,15 @@
 .TP
 .B \-showusage, \-su
 Displays a summary of all the command line options. For options that
-are not mentioned here, the short summary given by
-\fBmame \-showusage\fR is usually sufficient.
+are not mentioned here, the short summary given by \fB\-showusage\fR
+is usually sufficient.
 .\"
 .\" *******************************************************
 .SS Frontend commands
 .\" *******************************************************
-.B NOTE:
-By default, all the '\-list' commands below write info to the screen.
-If you wish to write the info to a textfile instead, use redirection, e.g.
+All the list commands below write info to the screen.
+If you wish to write the info to a textfile instead, use redirection.
+For example,
 .B mame \-listxml > ~/mamelist.xml
 writes the full list of supported game to file \fImamelist.xml\fR in your home
 directory.
@@ -155,8 +155,14 @@
 .B \-listsoftware, \-lsoft
 Output the list of known software for the system.
 .TP
+.B \-verifysoftware, \-vsoft
+Verify known software for the system.
+.TP
 .B \-getsoftlist, \-glist
 Retrieve software list by name.
+.TP
+.B \-verifysoftlist, \-vlist
+Verify software list by name.
 .\"
 .\" *******************************************************
 .SS Configuration options
@@ -204,11 +210,11 @@
 .\" *******************************************************
 .SS Search path options
 .\" *******************************************************
-.B IMPORTANT\fR: Please use the path, directory and file options in
+Be careful to use the path, directory and file options in
 mame.ini \fBONLY\fR. Otherwise, the outcome may be unpredictable and not
 consistent across releases.
 .TP
-.B \-rompath, \-rp, \-biospath, \-bp, \fIpathname
+.B \-rompath, \-rp, \-biospath, \-bp \fIpathname
 Specifies a list of paths within which to find ROM or hard disk images.
 Multiple paths can be specified by separating them with semicolons.
 The default is 'roms' (that is, a directory "roms" in the same directory
@@ -245,7 +251,7 @@
 .B \-cheatpath \fIpathname
 Specifies a list of paths within which to find cheat files. Multiple
 paths can be specified by separating them with semicolons. The default
-is 'cheat' (that is, a directory "cheat" in the same directory as
+is 'cheat' (that is, a directory 'cheat' in the same directory as
 the MAME executable).
 .TP
 .B \-crosshairpath \fIpathname
@@ -365,7 +371,7 @@
 and names the snapshots under it starting with 0000 and increasing
 from there.
 .TP
-.B \-snapsize \fIwidth\fPx\fIheight\fP
+.B \-snapsize \fIwidth\fPx\fIheight
 Hard\-codes the size for snapshots and movie recording. By default,
 MAME will create snapshots at the game's current resolution in raw
 pixels, and will create movies at the game's starting resolution in
@@ -374,7 +380,7 @@
 the result. Note that this size does not automatically rotate if the 
 game is vertically oriented. The default is 'auto'.
 .TP
-.B \-snapview \fIviewname\fP
+.B \-snapview internal\fR|\fBauto\fR|\fIviewname
 Specifies the view to use when rendering snapshots and movies. By
 default, both use a special 'internal' view, which renders a separate
 snapshot per screen or renders movies only of the first screen. By
@@ -505,9 +511,7 @@
 horizontally or vertically in one of the directions you can handle.
 If your screen only pivots counter\-clockwise, use \-autoror.
 .TP
-.B \-[no]flipx
-.TP
-.B \-[no]flipy
+.B \-[no]flipx \-[no]flipy
 Flip (mirror) the game screen either horizontally (\-flipx) or
 vertically (\-flipy). The flips are applied after the \-rotate and
 \-ror/\-rol options are applied. The default for both of these options
@@ -621,9 +625,6 @@
 .br
 Default is SOFT.
 .TP
-.B \-numscreens
-Reserved for future use.
-.TP
 .B \-[no]window, \-[no]w
 Run MAME in either full screen or a window. This is a fully\-featured window
 mode where the window resizes as necessary to track what the game does.
@@ -730,7 +731,7 @@
 .B \-gl_glsl
 Enable OpenGL GLSL, if available, for a performance increase.
 .TP
-.B \-gl_glsl_filter
+.B \-gl_glsl_filter \fIvalue
 Enable OpenGL GLSL filtering instead of FF filtering 0=plain, 1=bilinear.
 Default is 1: bilinear.
 .TP
@@ -795,7 +796,7 @@
 .B \-[no]sound
 Enable or disable sound altogether. The default is ON (\-sound).
 .TP
-.B \-samplerate, \-sr \fIvalue
+.B \-samplerate, \-srf \fIvalue
 Sets the audio sample rate. Smaller values (e.g. 11025) cause lower
 audio quality but faster emulation speed. Higher values (e.g. 48000)
 cause higher audio quality but slower emulation speed. The default is
@@ -929,8 +930,8 @@
 \fB\-keymap_file\fR.
 .TP
 .B \-keymap_file \fIkeymap_file\fR
-Specifies the full path to the keymap file to be used. Shipped keymap files lie
-in \fB/usr/share/games/mame/keymaps\fR.
+Specifies the full path to the keymap file to be used. A few
+keymap files are available in \fB/usr/share/games/mess/keymaps\fR.
 .TP
 .B \-joy_idx[0\-8] \fIjoystick
 With these options you can assign a joystick to a 
@@ -942,6 +943,9 @@
 .B \-sixaxis
 Use special handling for PS3 Sixaxis controllers.
 .TP
+.B \-lightgun_index[1\-8]
+Map lightgun to specific index in MAME.
+.TP
 .B \-videodriver, \-vd \fIx11\fR|\fIdirectfb\fR|\fIauto\fR
 SDL video driver to use; auto selects SDL default.
 .TP
@@ -1045,6 +1049,9 @@
 .TP
 .B \-confirm_quit
 Display confirm quit screen on exit.
+.TP
+.B \-ui_mouse
+Display UI mouse cursor.
 .\" +++++++++++++++++++++++++++++++++++++++++++++++++++++++
 .\" SDL specific
 .\" +++++++++++++++++++++++++++++++++++++++++++++++++++++++
Index: mame-0.147u3/src/osd/sdl/man/chdman.1
===================================================================
--- mame-0.147u3.orig/src/osd/sdl/man/chdman.1	2012-09-18 22:35:07.000000000 +0200
+++ mame-0.147u3/src/osd/sdl/man/chdman.1	2012-12-02 02:12:13.000000000 +0100
@@ -4,9 +4,9 @@
 .\"
 .\" Man page created from source and usage information by 
 .\" Ashley T. Howes <debiandev@ashleyhowes.com>, February 2005
-.\" updated by Cesare Falco <cesare.falco@gmail.com>, February 2007
+.\" updated by Cesare Falco <c.falco@ubuntu.com>, February 2007
 .\"
-.TH CHDMAN 1 2012-03-25 0.145u5 "MAME Compressed Hunks of Data (CHD) manager"
+.TH CHDMAN 1 2012-12-01 0.147u3 "MAME Compressed Hunks of Data (CHD) manager"
 .\"
 .\" NAME chapter
 .SH NAME
@@ -25,41 +25,154 @@
 .\"
 .\" OPTIONS chapter
 .SH OPTIONS
-.TP
-.B \info \fIinput\fR.chd
+Please note that each option allows a set of switches, the meaning of each
+is explained in the \fBSWITCHES\fR section below. Whenever the short form
+for the switch is listed, the long one may be used instead.
+.TP
+.B help \fR[\fIoption\fR]
+Print a summary of the available options or, if \fIoption\fR is given, a
+detailed list of the valid switches for the option.
+.TP
+.B info \
+\-i \fIfilename\fR \
+[\fB\-v\fR]
 Dump the header information from a drive image.
 .TP
-.B verify \fIinput\fR.chd
+.B verify \
+\-i \fIfileiname\fR \
+[\fB\-ip \fIfilename\fR]
 Validate the MD5/SHA1 on a drive image.
 .TP
-.B createraw \fIinputhd\fR.raw \fIoutput\fR.chd [\fIinputoffs\fR [\fIhunksize\fR]]
+.B createraw \
+\-o \fIfilename\fR \
+[\fB\-op \fIfilename\fR] \
+[\fB\-f\fR] \
+\fB\-i \fIfilename\fR \
+[\fB\-isb \fIoffset\fR] \
+[\fB\-ish \fIoffset\fR] \
+[\fB\-ib \fIlength\fR] \
+[\fB\-ih \fIlength\fR] \
+\fB\-hs \fIbytes\fR \
+\fB\-us \fIbytes\fR \
+[\fB\-c none\fR|type1[,[...]]] \
+[\fB\-np \fIprocessors\fR]
 Create a new compressed raw image from a raw file.
 .TP
-.B createhd \fIinputhd\fR.raw \fIoutput\fR.chd [\fIinputoffs\fR [\fIcylinders \
-heads sectors \fR[\fIsectorsize\fR [\fIhunksize\fR]]]]
+.B createhd \
+\-o \fIfilename\fR \
+[\fB\-op \fIfilename\fR] \
+[\fB\-f\fR] \
+[\fB\-i \fIfilename\fR] \
+[\fB\-isb \fIoffset\fR] \
+[\fB\-ish \fIoffset\fR] \
+[\fB\-ib \fIlength\fR] \
+[\fB\-ih \fIlength\fR] \
+[\fB\-hs \fIbytes\fR] \
+[\fB\-c none\fR|type1[,[...]]] \
+[\fB\-chs \fIcylinders\fB,\fIheads\fB,\fIsectors\fR] \
+[\fB\-ss \fIbytes\fR] \
+[\fB\-np \fIprocessors\fR]
 Create a new compressed hard disk image from a raw file.
 .TP
-.B createcd \fIinput\fR.toc \fIoutput\fR.chd
+.B createcd \
+\-o \fIfilename\fR \
+[\fB\-op \fIfilename\fR] \
+[\fB\-f\fR] \
+\fB\-i \fIfilename\fR \
+[\fB\-hs \fIbytes\fR] \
+[\fB\-c none\fR|type1[,[...]]] \
+[\fB\-np \fIprocessors\fR]
 Create a new compressed CD image from a raw file.
 .TP
-.B createld \fIinput\fR.toc \fIoutput\fR.chd
+.B createld \
+\-o \fIfilename\fR \
+[\fB\-op \fIfilename\fR] \
+[\fB\-f\fR] \
+\fB\-i \fIfilename\fR \
+[\fB\-isf \fIoffset\fR] \
+[\fB\-if \fIlength\fR] \
+[\fB\-hs \fIbytes\fR] \
+[\fB\-c none\fR|type1[,[...]]] \
+[\fB\-np \fIprocessors\fR]
 Create a new compressed laserdisc image from a raw file.
 .TP
-.B extractraw \fIinput\fR.chd \fIoutput\fR.raw
+.B extractraw \
+\-o \fIfilename\fR \
+[\fB\-f\fR] \
+\fB\-i \fIfilename\fR \
+[\fB\-ip \fIfilename\fR] \
+[\fB\-isb \fIoffset\fR] \
+[\fB\-ish \fIoffset\fR] \
+[\fB\-ib \fIlength\fR] \
+[\fB\-ih \fIlength\fR]
 Extract a raw file from a CHD image.
 .TP
-.B extracthd \fIinput\fR.chd \fIoutput\fR.toc \fIoutput\fR.bin
+.B extracthd \
+\-o \fIfilename\fR \
+[\fB\-f\fR] \
+\fB\-i \fIfilename\fR \
+[\fB\-ip \fIfilename\fR] \
+[\fB\-isb \fIoffset\fR] \
+[\fB\-ish \fIoffset\fR] \
+[\fB\-ib \fIlength\fR] \
+[\fB\-ih \fIlength\fR]
 Extract a hard disk block image from a CHD image.
 .TP
-.B extractcd \fIinput\fR.chd \fIoutput\fR.toc \fIoutput\fR.bin
+.B extractcd \
+\-o \fIfilename\fR \
+[\fB\-ob \fIfilename\fR] \
+[\fB\-f\fR] \
+\fB\-i \fIfilename\fR \
+[\fB\-ip \fIfilename\fR]
 Extract a CDRDAO .toc/.bin, CDRWIN .bin/.cue, or Sega Dreamcast .GDI file from a CHD\-CD image.
 .TP
-.B extractld \fIinput\fR.chd \fIoutput\fR.toc \fIoutput\fR.bin
+.B extractld \
+\-o \fIfilename\fR \
+[\fB\-f\fR] \
+\fB\-i \fIfilename\fR \
+[\fB\-ip \fIfilename\fR] \
+[\fB\-isf \fIoffset\fR] \
+[\fB\-if \fIlength\fR]
 Extract a laserdisc image from a CHD\-LD image.
 .TP
-.B copy \fIinput\fR.chd \fIoutput\fR.chd
+.B copy \
+\-o \fIfilename\fR \
+[\fB\-op \fIfilename\fR] \
+[\fB\-f\fR] \
+\fB\-i \fIfilename\fR \
+[\fB\-ip \fIfilename\fR] \
+[\fB\-isb \fIoffset\fR] \
+[\fB\-ish \fIoffset\fR] \
+[\fB\-ib \fIlength\fR] \
+[\fB\-ih \fIlength\fR] \
+\fB\-hs \fIbytes\fR \
+[\fB\-c none\fR|type1[,[...]]] \
+[\fB\-np \fIprocessors\fR]
 Copy all hunks of data from one CHD file to another. The hunk sizes do not need to match.
 If the source is shorter than the destination, the source data will be padded with 0s.
+.TP
+.B addmeta \
+\fB\-i \fIfilename\fR \
+\fB\-t \fItag\fR \
+[\fB\-ix \fIindex\fR] \
+[\fB\-vt \fItext\fR] \
+[\fB\-vf \fIfilename\fR] \
+[\fB\-nocs\fR]
+Add metadata to the chd.
+.TP
+.B delmeta \
+\fB\-i \fIfilename\fR \
+\fB\-t \fItag\fR \
+[\fB\-ix \fIindex\fR]
+Remove metadata from the chd.
+.TP
+.B dumpmeta \
+\fB\-i \fIfilename\fR \
+[\fB\-o \fIfilename\fR] \
+[\fB\-f\fR] \
+\fB\-t \fItag\fR \
+[\fB\-ix \fIindex\fR]
+Dump metadata from the chd to standard output or to a file.
 .\"
 .\" SWITCHES chapter
 .SH SWITCHES
@@ -67,32 +180,29 @@
 .B \-\-chs, \-chs \fIcylinders,heads,sectors
 Specifies CHS geometry values for CHD harddisks.
 .TP
-.B \-\-compression, \-c \fInone\fR|\fItype1\fR[,\fItype2\fR[,...]]
+.B \-\-compression, \-c \fInone\fR|\fItype1\fR[,[...]]
 Which compression codecs to use (up to 4).
 .TP
 .B \-\-force, \-f
 Force overwriting an existing file.
 .TP
-.B \-\-help, \-h
-Shows the available switches for an option.
-.TP
 .B \-\-hunksize, \-hs \fIbytes
 Size of each unit, in hunks.
 .TP
 .B \-\-ident, \-id \fIfilename
 Name of ident file to provide CHS information.
 .TP
+.B \-\-index, \-ix \fIindex
+Indexed instance of the metadata tag.
+.TP
 .B \-\-input, \-i \fIfilename
 Input file name.
 .TP
-.B \-\-inputstartbyte, \-isb \fIoffset
-Starting byte offset within the input.
-.TP
-.B \-\-inputstarthunk, \-ish \fIoffset
-Starting hunk offset within the input.
-.TP
 .B \-\-inputbytes, \-ib \fIlength
 Effective length of the input in bytes.
+.TP
+.B \-\-inputframes, \-if \fIlength
+Effective length of the input in frames.
 .TP                              
 .B \-\-inputhunks, \-ih \fIlength
 Effective length of the input in hunks.
@@ -100,14 +210,20 @@
 .B \-\-inputparent, \-ip \fIfilename
 Parent CHD's input file name.
 .TP
-.B \-\-numprocessors, \-np \fIcount
-Limits the number of processors to use during compression.
+.B \-\-inputstartbyte, \-isb \fIoffset
+Starting byte offset within the input.
 .TP
-.B \-\-sectorsize, \-ss \fIbytes
-size of each hard disk sector.
+.B \-\-inputstartframe, \-isf \fIoffset
+Starting frame offset within the input.
 .TP
-.B \-\-verbose, \-v
-Output additional information about the operation.
+.B \-\-inputstarthunk, \-ish \fIoffset
+Starting hunk offset within the input.
+.TP
+.B \-\-nochecksum, \-nocs
+Do not include this metadata information in the overall SHA-1.
+.TP
+.B \-\-numprocessors, \-np \fIcount
+Limits the number of processors to use during compression.
 .TP
 .B \-\-output, \-o \fIfilename
 Output file name.
@@ -118,6 +234,20 @@
 .B \-\-outputparent, \-op \fIfilename
 Parent CHD's output file name.
 .TP
+.B \-\-sectorsize, \-ss \fIbytes
+Size of each hard disk sector.
+.TP
+.B \-\-tag, \-t \fItag
+4-character tag for metadata.
+.TP
+.B \-\-valuefile, \-vf \fIfilename
+Text file containing data to add.
+.TP
+.B \-\-valuetext, \-vt \fItext
+Text for the metadata.
+.TP
 .B \-\-unitsize, \-us \fIbytes
 Size of each unit, in bytes.
-
+.TP
+.B \-\-verbose, \-v
+Output additional information about the operation.
Index: mame-0.147u3/src/osd/sdl/man/jedutil.1
===================================================================
--- mame-0.147u3.orig/src/osd/sdl/man/jedutil.1	2012-09-18 22:35:07.000000000 +0200
+++ mame-0.147u3/src/osd/sdl/man/jedutil.1	2012-12-02 02:12:13.000000000 +0100
@@ -3,12 +3,12 @@
 .\" jedutil.1
 .\"
 .\" Man page created from source and usage information
-.\" Cesare Falco <cesare.falco@gmail.com>, February 2007
+.\" Cesare Falco <c.falco@ubuntu.com>, February 2007
 .\" 
 .\" References
 .\" http://aarongiles.com/?p=159
 .\"
-.TH JEDUTIL 1 2012-03-25 0.145u5 "jedutil"
+.TH JEDUTIL 1 2012-12-01 0.147u3 "MAME JED files manager"
 .\"
 .\" NAME chapter
 .SH NAME
@@ -21,6 +21,8 @@
 .br
 .B jedutil
 .IR source ".bin " target .jed
+.br
+.B jedutil help
 .\"
 .\" DESCRIPTION chapter
 .SH DESCRIPTION
Index: mame-0.147u3/src/osd/sdl/man/ldverify.1
===================================================================
--- mame-0.147u3.orig/src/osd/sdl/man/ldverify.1	2012-09-18 22:35:07.000000000 +0200
+++ mame-0.147u3/src/osd/sdl/man/ldverify.1	2012-12-02 02:12:13.000000000 +0100
@@ -3,9 +3,9 @@
 .\" ldverify.1
 .\"
 .\" Man page created from source and usage information by 
-.\" Cesare Falco <cesare.falco@gmail.com>, August 2008
+.\" Cesare Falco <c.falco@ubuntu.com>, August 2008
 .\"
-.TH LDVERIFY 1 2012-03-25 0.145u5 "ldverify"
+.TH LDVERIFY 1 2012-12-01 0.147u3 "MAME laserdisc data checker"
 .\"
 .\" NAME chapter
 .SH NAME
Index: mame-0.147u3/src/osd/sdl/man/romcmp.1
===================================================================
--- mame-0.147u3.orig/src/osd/sdl/man/romcmp.1	2012-09-18 22:35:07.000000000 +0200
+++ mame-0.147u3/src/osd/sdl/man/romcmp.1	2012-12-02 02:12:13.000000000 +0100
@@ -4,12 +4,12 @@
 .\"
 .\" Man page created from FAQs, source and usage information by 
 .\" Ashley T. Howes <debiandev@ashleyhowes.com>, February 2005
-.\" updated by Cesare Falco <cesare.falco@gmail.com>, February 2007
+.\" updated by Cesare Falco <c.falco@ubuntu.com>, February 2007
 .\" 
 .\" References
 .\" http://www.mame.net/mamefaq.html
 .\"
-.TH ROMCMP 1 2012-03-25 0.145u5 "romcmp"
+.TH ROMCMP 1 2012-12-01 0.147u3 "MAME romset checking tool"
 .\"
 .\" NAME chapter
 .SH NAME
Index: mame-0.147u3/src/osd/sdl/man/testkeys.1
===================================================================
--- mame-0.147u3.orig/src/osd/sdl/man/testkeys.1	2012-09-18 22:35:07.000000000 +0200
+++ mame-0.147u3/src/osd/sdl/man/testkeys.1	2012-12-02 02:12:13.000000000 +0100
@@ -3,9 +3,9 @@
 .\" testkeys.1
 .\"
 .\" Man page created from source and usage information
-.\" Cesare Falco <cesare.falco@gmail.com>, February 2007
+.\" Cesare Falco <c.falco@ubuntu.com>, February 2007
 .\"
-.TH TESTKEYS 1 2012-03-25 0.145u5 "testkeys"
+.TH TESTKEYS 1 2012-12-01 0.147u3 "MAME SDL keycode checker"
 .\"
 .\" NAME chapter
 .SH NAME
Index: mame-0.147u3/src/osd/sdl/man/ldplayer.1
===================================================================
--- mame-0.147u3.orig/src/osd/sdl/man/ldplayer.1	2012-09-18 22:35:07.000000000 +0200
+++ mame-0.147u3/src/osd/sdl/man/ldplayer.1	2012-12-02 02:12:43.000000000 +0100
@@ -3,7 +3,7 @@
 .\" ldplayer.1
 .\"
 .\" Man page created from source and usage information by 
-.\" Cesare Falco <cesare.falco@gmail.com>, August 2008
+.\" Cesare Falco <c.falco@ubuntu.com>, August 2008
 .\"
 .TH LDPLAYER 1 2012-03-25 0.145u5 "ldplayer"
 .\"
